### DISCUSSION TITLE
Add option to prune old docker image after update

### DISCUSSION THREAD

**Discussion by MrAvana** [louislam/dockge#225] [Feature Request]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

UI Feature

### 🔖 Feature description

There should be a settings option to prune old docker images after I update a stack.

### ✔️ Solution

An option in the settings menu would be great.

**Comment by trythatagain** [comment]
Also perhaps a menu called prune, with drop down menu with images, networks, containers, volumes and all choices

**Reply by lephtHanded** [reply]
I like this idea as it would allow me to prune all the unused images, not just the ones in the stack that I'm in.

**Reply by xtr3-m3** [reply]
Yes, i am also missing this feature. +1 from me! Would be nice!
